1_10/pom.xml

```diff
@@ -10,7 +10,7 @@
         <version>1.9.3-SNAPSHOT</version>
     </parent>
 
-    <artifactId>anvilgui-1_10_R1</artifactId>
+    <artifactId>anvilgui-1_10_2</artifactId>
```

Comment:
The `artifactId` should be `1_10` instead of `1_10_2`, to match the module and Wrapper class name